src/hooks/useBreakpoints.ts

```diff
@@ -3,6 +3,7 @@ import { useEffect, useState } from 'react'
 const xsSize = 279
 const smSize = 375
 const mdSize = 600
+const tailwindMd = 767
```

Comment:
I can't get why it's called `tailwindMd`? What does it mean? Is there another suitable name for it?